cli/cmd/policy_disable.go

```diff
@@ -0,0 +1,71 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2023, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/aws/smithy-go/ptr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// This is an experimental command.
+	// policyDisableTagCmd represents the policy disable command
+	policyDisableTagCmd = &cobra.Command{
+		Use:   "disable [policy_id]",
```

Comment:
We have NOT had this before where a command accepts a number of arguments of the same type, the help message should help the user understand that, let us standardize in this format:
```suggestion
		Use:   "disable [policy_id...]",
```

Please update every command.